src/tapis-ui/components/jobs/JobLauncher/FileInputs.tsx

```diff
@@ -108,6 +108,7 @@ const FileInputs: React.FC<FileInputsProps> = ({ appInputs }) => {
     render: FileInputField,
     required,
     appendData,
+    isCollapsable: required.length > 0,
```

Comment:
If required.length > 0 is it no longer collapsable?

Reply:
No collapse UI will be rendered if there are 0 required file inputs